docs/cmdlets-reference/restore-cohesityremotefile.md

```diff
@@ -0,0 +1,269 @@
+# Restore-CohesityRemoteFile
+
+## SYNOPSIS
+Restores the specified files or folders from a previous backup from a remote cluster.
+
+## SYNTAX
+
+```
+Restore-CohesityRemoteFile [[-TaskName] <String>] [-FileNames] <String[]> [-JobId] <Int64> [-SourceId] <Int64>
+ [[-NewBaseDirectory] <String>] [[-JobRunId] <Int64>] [[-StartTime] <Int64>] [-DoNotOverwrite]
+ [-ContinueOnError] [-DoNotPreserveAttributes] [-TargetSourceId] <Int64>
+ [[-TargetHostCredential] <PSCredential>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Restores the specified files or folders from a previous backup from a remote cluster.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Restore-CohesityRemoteFile -TaskName "restore-file-vm" -FileNames /C/data/file.txt -JobId 1234 -SourceId 843 -TargetSourceId 856 -TargetParentSourceId 828 -TargetHostType KWindows -TargetHostCredential (Get-Credential)
+```
+
+Restores the specified file to the target windows VM with the source id 843 from the latest backup.
+
+### EXAMPLE 2
+```
+Restore-CohesityRemoteFile  -FileNames "/C/myFolder" -NewBaseDirectory "C:\temp\restore" -JobId 61592 -SourceId 3517 -TargetSourceId 3098
```

Comment:
Why is the forward and backward slash inconsistent in the same cmdlet ? 

If taking IDs , can you make the example more helpful by getting those cmdlets from other cmdlets and assigning IDs to a variable.

Reply:
The payload accepts filenames and newbasedirectory in the given format only.
OK, will add examples, how to extract the IDs.

Reply:
Done. Added example how to extract ids.